database/mssql/scripts/dbscripts/PSP_PIMS_S15_00/Test Data/03_DML_LEASE_TESTING.sql

```diff
@@ -7,28 +7,45 @@ Doug Filteau  2021-Aug-19  Initial version
 ----------------------------------------------------------------------------- */
 
 -- Insert addresses
-INSERT INTO PIMS_ADDRESS (STREET_ADDRESS_1, STREET_ADDRESS_2, STREET_ADDRESS_3, MUNICIPALITY_NAME, POSTAL_CODE, PROVINCE_STATE_ID, COUNTRY_ID, ADDRESS_USAGE_TYPE_CODE, APP_CREATE_TIMESTAMP, APP_CREATE_USERID, APP_CREATE_USER_DIRECTORY, APP_LAST_UPDATE_TIMESTAMP, APP_LAST_UPDATE_USERID, APP_LAST_UPDATE_USER_DIRECTORY)
```

Comment:
I saw your comment on the other PR about including this in the build/alter scripts, I don't think we want to do that since this is strictly test data. My way of looking at this:
Build/alter folders -> can run in all environments, including prod
test data folders -> can run locally, and in dev/test environments